backend/lcfs/web/api/fuel_code/repo.py

```diff
@@ -798,12 +792,29 @@ async def get_standardized_fuel_data(
         if target_carbon_intensities:
             target_ci = next(
                 (tci.target_carbon_intensity for tci in target_carbon_intensities),
-                0,
+                0.0,
             )
 
-        return {
-            "effective_carbon_intensity": effective_carbon_intensity,
-            "target_ci": target_ci,
-            "eer": eer,
-            "energy_density": energy_density,
-        }
+        # Additional Carbon Intensity (UCI)
+        uci = await self.get_additional_carbon_intensity(fuel_type_id, end_use_id)
+
+        return CarbonIntensityResult(
+            effective_carbon_intensity=effective_carbon_intensity,
+            target_ci=target_ci,
+            eer=eer,
+            energy_density=energy_density,
+            uci=uci.intensity if uci else None,
+        )
+
+    @repo_handler
+    async def get_additional_carbon_intensity(
+        self, fuel_type_id, end_use_type_id
+    ) -> Optional[AdditionalCarbonIntensity]:
+        """Get a single use of a carbon intensity (UCI), returns None if one does not apply"""
+        query = select(AdditionalCarbonIntensity).where(
+            AdditionalCarbonIntensity.end_use_type_id == end_use_type_id,
```

Comment:
End use is always "Any" in endpoint /compliance-reporting/:compliancePeriod/:complianceReportId/supply-of-fuel . It is not possible to check how the uci is changing when the user selects different end use. Based on the code should work, but it would be nice to check in UI.

Reply:
You are correct that currently there are no possible fuel combinations that use UCI. I modified by own fuel codes to properly test.
![image](https://github.com/user-attachments/assets/72306725-f751-4c9a-bdb6-8b6ad7da45b6)